lib/cmds/space_cmds/use.js

```diff
@@ -24,32 +24,31 @@ export const builder = (yargs) => {
     .epilog('Copyright 2018 Contentful, this is a BETA release')
 }
 
-function showSuccess (space) {
-  success(`Now using the 'master' Environment of Space ${highlightStyle(space.name)} (${highlightStyle(space.sys.id)}) when the \`--environment-id\` option is missing.`)
+function showSuccess (space, env) {
+  success(`Now using the '${env}' Environment of Space ${highlightStyle(space.name)} (${highlightStyle(space.sys.id)}) when the \`--environment-id\` option is missing.`)
```

Comment:
shouldn't have single quotes around it, right? otherwise the string interpolation won't work?

Reply:
I would think this still works, as the quotes (single or double) are handled as normal strings within backticks .. Am i mistaken ?

Reply:
yeah you're right my bad 👍